CMakeLists.txt

```diff
@@ -373,6 +347,41 @@ if (NOT S2N_EXECINFO_AVAILABLE)
 endif()
 feature_probe_result(S2N_STACKTRACE ${S2N_STACKTRACE})
 
+if(S2N_BRANCH_ALIGN AND NOT S2N_FUZZ_TEST)
+    set(BRANCH_ALIGN_FLAG "-Wa,-mbranches-within-32B-boundaries")
+    message(STATUS "Feature probe succeeded: S2N_BRANCH_ALIGN is supported.")
+else()
+    set(BRANCH_ALIGN_FLAG "")
+    message(STATUS "Feature probe failed: S2N_BRANCH_ALIGN is not supported.")
+endif()
+
+target_compile_options(${PROJECT_NAME} PRIVATE 
```

Comment:
Is moving this down here necessary? My understanding is that target_compile_options is additive, so you could call it for just the new arguments.